molr-mole-core/src/main/java/io/molr/mole/core/runnable/lang/RunnableLeafsMissionSupport.java

```diff
@@ -50,6 +56,14 @@ private void assertNoBuilderYet() {
             throw new IllegalStateException("Root can only be defined once! Use either sequential() or parallel().");
         }
     }
+    
+    protected OngoingExecutionStrategyConfiguration executionStrategy() {
+    	if(executionStrategyConfigurationBuilder!=null) {
```

Comment:
Potentially one would need a different logic here ... 
It should be allowed that executionStrategy() is never called (Which should result in the default).
What would happen in this case?
(Could be solved by always having a builder, with an additional flag)